### DISCUSSION TITLE
Deleting comments?

### DISCUSSION THREAD

**Discussion by Kryptortio** [jgraph/drawio#2582] [General]
I just thought it was a bit strange that my comment on [this issue](https://github.com/jgraph/drawio/issues/711) was deleted. My suggestion was:

> This would be great, shouldn't be a lot of work to just add a text field to the base model. It would be good though if there was a small visible icon on all items with a note in the diagram so you can easily see that they exists. Also this icon could be used to show the note on a mouseover popup.

Maybe the first sentence is kind of useless but is a visual indicator such a bad idea that the comment needs to be deleted? Wouldn't it be better to just reply something like "I don't think a visual indicator would be good because ..." or even just not implementing that part if you don't like it but leaving the comment for posterity? Leaving it even if you don't like it might also inspire a variation of it that you do like e.g. also adding a checkbox to make it optional.

**Comment by davidjgraph** [comment]
"Shouldn't be a lot of work" is the problem. Everything is a lot of work, users with no idea of the amount of trying to guilt devs into implementing something by implying "this is simple, it will take you 30 minutes" 1) have no idea how long software engineering actually takes and 2) can implement it themselves if it's so easy.

That aside, you're welcome to add the comment without that part.

**Comment by Kryptortio** [comment]
Right, I'll do that, it's a shame losing feedback because of issues like that though. Maybe consider copying the rest of the comment before deleting it or just asking the user to please avoid writing things like that. In this case it was just luck that I noticed it had happened at all. If someone doesn't notice and ask about it they might just keep doing it (not knowing that it was deleted or why). 

As for this case specifically, sorry if I offended I truly meant nothing by it. It was just me as a developer thinking about the solution (without really knowing anything about it). I can see how it can appear pushy when I just wanted to provide some feedback.
